env/dev/kustomization.yaml

```diff
@@ -677,10 +677,3 @@ vars:
     apiVersion: v1
     kind: ConfigMap
     name: application-config
-- fieldref:
```

Comment:
You meant to take out XRAY for dev?

Reply:
ya, that was old code that had no business being there.